jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/JKubeServiceHub.java

```diff
@@ -71,16 +70,9 @@ public JKubeServiceHub(
     private void init() {
         Objects.requireNonNull(configuration, "JKubeKitConfiguration is required");
         Objects.requireNonNull(log, "log is a required parameter");
-        if (platformMode == null) {
```

Comment:
If platformMode is now required, add the nullcheck:
`Objects.requireNonNull(platformMode, "platformMode is a required parameter");`